luma_core/src/ipc.rs

```diff
@@ -1,26 +1,26 @@
-use core::ptr::from_exposed_addr_mut;
+use core::ptr;
 
 //bits 0..=31 = physical address of ipc request
-const HW_IPC_PPCMSG: usize = 0xCD00_0000usize; //from_exposed_addr_mut(0xCD00_0000);
+const HW_IPC_PPCMSG: *mut u32 = ptr::without_provenance_mut(0xCD00_0000);
```

Comment:
we want `with_provenance_mut` here instead of `without_provenance_mut` otherwise writing to value is UB